crates/adapters/src/transport/secret_resolver.rs

```diff
@@ -43,8 +43,8 @@ impl Debug for MaybeSecret {
 }
 
 impl MaybeSecret {
-    /// Path to the default secret volume.
-    pub const DEFAULT_SECRET_VOLUME_PATH: &'static str = "/etc/secret-volume";
+    /// Path of the default secrets directory.
+    pub const DEFAULT_SECRETS_DIRECTORY_PATH: &'static str = "/etc/secrets";
```

Comment:
Could we make this a configuration option for the pipeline itself (i.e., in `PipelineConfig`)? This might run into permissions issues on the local form factor.

Reply:
I agree configuration at pipeline-level would be preferable, which would make secrets a more direct part of the API. We can do it in a separate PR, in order to keep the scope of this PR limited as an minor update to the first secret management PR.